governance.md

```diff
@@ -0,0 +1,340 @@
+in terms of project direction, to potentially less experienced contributors.
+
+The Steering Council and its Members play a special role in certain situations.
+In particular, the Council may:
+
+-   Make decisions about the overall scope, vision and direction of the
+    project.
+-   Make decisions about strategic collaborations with other organizations or
+    individuals.
+-   Make decisions about specific technical issues, features, bugs and pull
+    requests. They are the primary mechanism of guiding the code review process
+    and merging pull requests.
+-   Make decisions about the Services that are run by The Project and manage
+    those Services for the benefit of the Project and Community.
+-   Make decisions when regular community discussion doesn’t produce consensus
+    on an issue in a reasonable time frame.
+
+### Council membership
+
+To become eligible for being a Steering Council Member an individual should
+
+-  Be a Project Contributor who has produced contributions that are substantial in
+quality and quantity
+-  Sustain this development consistently over several months
+-  Support the community through code review, answering user questions,
+   triaging bug reports, participating constructively in broader conversations,
+   and being willing to help with documentation and infrastructure.
+-  Demonstrate breadth by supporting the community outside of their particular
+   work interests or sub-project
+-  Be civil in public discourse
+
+Potential Council Members are nominated by existing Council members and voted
+upon by the existing Council after asking if the potential Member is interested
+and willing to serve in that capacity. The Council will be initially formed
+from the set of existing Core Developers who, as of 2019, have ownership rights
+over the organization.
+
+If a Council member becomes inactive in the project for a period of one year,
+they will be considered for removal from the Council. Before removal, the
+inactive Member will be approached to see if they plan to return
+to active participation. If not, they will be removed immediately upon a Council
+vote. If they plan to return to active participation soon, they will be
+given a grace period of one year. If they don’t return to active participation
+within that time period they will be removed by vote of the Council without
+further grace period. All former Council members can be considered for
+membership again at any time in the future, like any other Project Contributor.
+Retired Council members will be listed on the project website, acknowledging
+the period during which they were active in the Council.
+An inactive BDFL may also be ejected by a consensus of current members under
+these same conditions.
+
+The Council reserves the right to eject current Members, other than the BDFL,
+if they are deemed to be actively harmful to the project’s well-being, and
+attempts at communication and conflict resolution have failed.
```

Comment:
Would be useful to have some percentage here for removing members as well.

Reply:
I would follow the standard system here.  If you have suggestions otherwise then please voice them.

Reply:
As there seem to be two threads about ejecting members, I'd suggest we consolidate them into one thread. Would arbitrarily suggest this one and we can apply whatever we decide for both cases (assuming that sounds agreeable).

For other decisions following the standard system (am guessing this is what is outlined in the new commit?) seems fine. When it comes to removing members though, I think the bar should be a bit higher. Would suggest a large majority (in conda-forge it's 75%). Also would suggest that not voting in this case does not count as approval given the nature of the decision.

While I've made this suggestion, would be happy to discuss further if there are other thoughts on this.

Reply:
> Would suggest a large majority (in conda-forge it's 75%). Also would suggest that not voting in this case does not count as approval given the nature of the decision.

Historically I haven't been able to get this kind of quorum among current organization owners.  I think that, at least with current engagement, this removes the possibility of ejecting people.

Reply:
I don't think that we should prescribe too much here.  The lazy consensus model seems to mostly work.  We can tweak things if necessary in the future.  I would prefer to see us add details only as necessary.  (though I acknowledge that, in cases like these, it's good to have foresight).

Reply:
> As there seem to be two threads about ejecting members, I'd suggest we consolidate them into one thread

Happy to consolidate things.  I'm not sure what the other thread is though.  Are you pointing to the paragraph just above this one about inactivity?

Reply:
> Happy to consolidate things. I'm not sure what the other thread is though. Are you pointing to the paragraph just above this one about inactivity?

Yes. Continuing here seems fine.

Reply:
I’m also fine with the lazy consensus.

Reply:
Matt and I talked about this offline. I'm going to write up some proposed text and push it to the PR so we can review together. Happy to iterate from there.

Reply:
I've pushed some text that provides some guidance on when this may be appropriate to use and when not. Please take a look and let me know your thoughts.

Reply:
Also have pushed some text borrowed from the membership document for additions and applied it to this removal section. Hopefully that strikes the balance between handling this carefully and not being overly complex. Please let me know if you have thoughts on that as well.